### ISSUE TITLE
HiveMQ Cloud

### ISSUE THREAD

**Issue by florianru1984** [Pmant/ioBroker.mqtt-client#61]
I tried to use HiveMQ Cloud as MQTT Broker. 
But the connection fails everytime.

If i use MQTT.fx Desktop CLient i get connection to HiveMQ Cloud

**Comment by Apollon77** [comment]
Does the logs from that service state any reasone for it? Otherwise ... how should we support here?

**Comment by pos-ei-don** [comment]
Same here. Other Clients work. Snny suggestions?

**Comment by AxelRHD** [comment]
@Apollon77, do you need any further information? I would deliver all I can to fix the issue and use your product.

**Comment by Apollon77** [comment]
I'm not an MQTT expert, so I could just guess ... but the mail "issue" ith this "issue" is that I aksed for logs, but never saw a single log line

**Comment by AxelRHD** [comment]
Unfortunately `Connection refused: Not authorized` is the only log, ioBroker shows to me. Like I stated in Ocotber already I've tried different combinations regarding the ports and the notation of the address.

**Comment by Apollon77** [comment]
Guys ... If I ask for a debug log then please provide a debug log or you need to try to fix it yourself. A Log provides much more information then just a "part of one logline from somewhere". It is also about WHERE and WHEN it is logged in the initialization process and also more stuff. I do not say it is "the" thing, but it provides more context for a developer that find time to look into it -especially ideally without needing to repo it myself.

With these rare information I need to find out how to setup "HiveMQ Cloud" to try and repro all that myself ... and honestly ... if I need to do that then it is not very likely that I can look into it soon because of the "immense" time I would need for that compared to other prios in other adapters where fixes are having all provided information or affecting more then 3 users (sorry to say it that way).

Also: Why noone uses "thumbs up" on first post ... so it is completely invisible how many users are affected ... and so very hard to define a prio ... or I need to spend time to cound "me too" posts :-(

So to summarize: You asked which further informations I need? Yes, a debug log. Thank you.
